src/components/Users/UserResetPassword.tsx

```diff
@@ -225,7 +231,7 @@ export default function UserResetPassword({
               </Button>
               <Button
                 type="submit"
-                disabled={!form.formState.isDirty || isPending}
+                disabled={!allFieldsFilled || isPending}
```

Comment:
```suggestion
                disabled={!form.formstate.isValid || isPending}
```
this should validate the passwords